cmd/plume/fedora.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2018 Red Hat Inc.
```

Comment:
This entire `plume: Draft the initial specs for Fedora images` commit is moving files to a different location that were originally moved/created in the `plume: Move the Fedora Specs into a different file` commit. Why not just move them into the correct location in the original commit?

Reply:
I skipped this squash as there was a commit in between these two, which might lead to conflicts.

Reply:
This would be fixed when the commits would be squashed into 3